src/AutoRest.CSharp.V3/Generation/Writers/CodeWriter.cs

```diff
@@ -367,7 +367,7 @@ private void PopScope(CodeWriterScope expected)
             Debug.Assert(actual == expected);
         }
 
-        public void RemoveTrailingComma()
+        private void ActionOnLast(Action<int> action)
```

Comment:
Should we just have a method that finds index of last non whitespace char?

```
var lastNonSpace = FindLastNonSpace();

if (lastNonSpace != -1 && _builder[lastNonSpace] == '{') 
{
}
```

Reply:
Done.